lib/dolly/document.rb

```diff
@@ -1,9 +1,13 @@
 require "dolly/query"
 require "dolly/property"
+require 'dolly/timestamps'
+require 'active_model/callbacks'
 
 module Dolly
   class Document
     extend Dolly::Connection
+    extend ActiveModel::Callbacks
```

Comment:
My proposal is to have a class method like `timestamps!` that you can call on the class definition, to add them